src/tasks/AotCompilerTask/MonoAOTCompiler.props

```diff
@@ -15,7 +15,15 @@
     <MonoAOTCompilerDefaultProcessArguments Include="-O=gsharedvt" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetOS)' == 'Android'">
+    <MonoAOTCompilerDefaultAotArguments Condition="'$(TargetArchitecture)' == 'arm'" Include="mtriple=armv7-linux-gnueabi-thumbv7s" />
     <MonoAOTCompilerDefaultAotArguments Include="static" />
+    <!-- Default trampolines run out for libraries tests -->
```

Comment:
sounds like if we run into the limits ourselves we should probably bump the default?

Reply:
@vargaz ?

Reply:
Issue filed https://github.com/dotnet/runtime/issues/53033

Reply:
Out of curiosity, what are trampolines and how were these new values determined?

Reply:
@mdh1418 https://www.mono-project.com/docs/advanced/runtime/docs/trampolines/